### PR TITLE
feat: add suggested tag client

### LINKED ISSUE #26792: [Feature]: Client side changes for Template Suggestion
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

Adds suggested field for query templates.

As seen below:

262922769-45846aef-ed94-4ba7-87c3-bbfb5e113990

### Why should this be worked on?

This provides a better experience for users as part of the Activation project.

### PR DESCRIPTION
**Pull request by Irongade**
This PR adds suggested tags to datasource schema query templates

#### PR fixes following issue(s)
Fixes #26792

- New feature (non-breaking change which adds functionality)

- [x] Manual
- [ ] JUnit
- [ ] Jest
- [x] Cypress
>
>
#### Test Plan
https://github.com/appsmithorg/TestSmith/issues/2528

>
>
#### Dev activity
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

#### QA activity:
- [ ] [Speedbreak features](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#speedbreakers-) have been covered
- [x] Test plan covers all impacted features and [areas of interest](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#areas-of-interest-)
- [ ] Test plan has been peer reviewed by project stakeholders and other QA members
- [x] Manually tested functionality on DP
- [ ] We had an implementation alignment call with stakeholders post QA Round 2
- [x] Cypress test cases have been added and approved by SDET/manual QA
- [ ] Added `Test Plan Approved` label after Cypress tests were reviewed
- [ ] Added `Test Plan Approved` label after JUnit tests were reviewed

### CODE REVIEW

File: app/client/src/pages/Editor/Explorer/Datasources/QueryTemplates.tsx [hunk 1]
```diff
@@ -192,6 +195,10 @@ export function QueryTemplates(props: QueryTemplatesProps) {
     ],
   );
 
+  const transformText = (s: string) => {
```
Comment (sneha122) [inline app/client/src/pages/Editor/Explorer/Datasources/QueryTemplates.tsx:198]: @Irongade Can we move this function to utils? also can we please rename this to something like toCamelCase()? This way it becomes reusable

**Comment by btsgh** [comment]
Completed testing this on the DP. 
One [issue](https://github.com/appsmithorg/appsmith/issues/25136) exists in terms of firefox, for the schema add query on property pane, which is also present for the new schema on the query page right side. 
Other than that seems good to go.

**Review by sneha122 — APPROVED** [review]

**Pull request merged**